ui/src/shared/components/ConfirmButtons.js

```diff
@@ -1,11 +1,22 @@
 import React, {PropTypes} from 'react'
+import classnames from 'classnames'
 
-const ConfirmButtons = ({onConfirm, item, onCancel}) => (
+const ConfirmButtons = ({onConfirm, item, onCancel, buttonSize}) => (
   <div className="confirm-buttons">
-    <button className="btn btn-xs btn-info" onClick={() => onCancel(item)}>
+    <button
+      className={classnames('btn btn-info btn-square', {
+        [buttonSize]: buttonSize,
+      })}
```

Comment:
I wonder if this can be accomplished more simply. I think classnames can take an array. If present, it'll concatenate. If absent, it gets omitted?:

`className={classnames('btn btn-info btn-square', buttonSize)}`

Not 100% that works, but worth a shot